### DISCUSSION TITLE
Reboot Faikin via MQTT

### DISCUSSION THREAD

**Discussion by dahadd** [revk/xESP32-Faikout#593] [General]
Is there a way to reboot the Faikin via MQTT? I cannot access the web interface, but the device is online and can be controlled via MQTT. Thanks

**Comment by revk** [comment]
`command/Faikin/restart`

**Reply by dahadd** [reply]
Thanks REVK.
via MQTT Explorer oder Homeassitant?
sorry, i don't really know much about it. Can you please explain this to me in more detail?

**Reply by revk** [reply]
Err, an MQTT message with a topic ending `restart`

**Reply by dahadd** [reply]
sorry, I still don't know what you mean and how to send an MQTT message. I am unfortunately not that knowledgeable in this matter

**Reply by peter-forrest** [reply]
> Thanks REVK.
> 
> via MQTT Explorer oder Homeassitant?
> 
> sorry, i don't really know much about it. Can you please explain this to me in more detail?

Watch this, then you'll be an expert in ten minutes, it helped me understand MQTT and topics and publishing (turns out it's actually pretty simple):

https://youtu.be/dqTn-Gk4Qeo?si=JJbA8fh3zynVsE_Z

**Comment by revk** [comment]
Ok sorry you asked about an MQTT message.

Alternative way to reboot is simply power cycle it.

**Reply by dahadd** [reply]
Thank you, but I don't want to switch it on and off. I just need a little support on how to send an MQTT message via Homeassinstant.

**Reply by revk** [reply]
No idea how to do that via HA. I have various tools to send MQTT messages. I would hope HA has some, but no idea.

**Reply by dahadd** [reply]
I have found a solution in HA.
Can you please tell me the tools for sending MQTT messages?

Thank you and a happy new year.

**Reply by revk** [reply]
Err is uses firebrick console, my own mqtt command code, and possibly mosquitto command line.
